docs/model-dev-guide/api-guides/apis-howto/_index.rst

```diff
@@ -56,21 +56,10 @@ Prefer to use an Example Model?
 If you'd like to build off of an existing model that already runs on Determined, visit our
 :ref:`example-solutions` to see if the model you'd like to train is already available.
 
-******************
- AMD ROCm Support
-******************
+ROCm Support
```

Comment:
We seem to be inconsistent about how we're using the terms "ROCm" and "AMD ROCm". I would have thought it best to use "AMD ROCm" first in any given location and in headings and then use just "ROCm" in subsequent references for brevity. Thoughts?

Reply:
agree